dfdaemon/proxy/proxy.go

```diff
@@ -0,0 +1,117 @@
+package proxy
+
+import (
+	"fmt"
+	"io"
+	"net"
+	"net/http"
+	"time"
+
+	"github.com/dragonflyoss/Dragonfly/dfdaemon/config"
+	"github.com/dragonflyoss/Dragonfly/dfdaemon/handler"
+	"github.com/sirupsen/logrus"
```

Comment:
Could you please add a blank line here? THX 😄

Reply:
Added in the latest commit, but the the code snippet some how is not updated or marked outdated.